Hummingbird.docc/index.md

```diff
@@ -27,16 +27,12 @@ let app = Application(
 try await app.runService()
 ```
 
-### Hummingbird Extensions
-
-Hummingbird is designed to require the least number of dependencies possible, but this means many features are unavailable to the core libraries. Additional features are provided through extensions. The Hummingbird repository comes with additional modules 
+## Topics
 
-- ``HummingbirdJobs``: framework for pushing work onto a queue to be processed outside of a request (possibly by another server instance)
-- ``HummingbirdTLS``: TLS support.
-- ``HummingbirdHTTP2``: Support for HTTP2 upgrades.
-- ``HummingbirdTesting``: helper functions to aid testing Hummingbird projects.
+### Tutorials
 
-## Topics
+- <doc:GettingStarted>
+- <doc:Todos>
```

Comment:
Can you do the same in Hummingbird.md. Bring the tutorial to the top and add the getting started link